### ISSUE TITLE
[Suggestion] Add variable item count to recipe output

### ISSUE THREAD

**Issue by Kevadroz** [Low-Drag-MC/Multiblocked2#50]
The title is pretty much self explanatory.

Yeah, basically make it so you can set a minimum and a maximum amount of a single item to give on recipe outputs.

Or maybe in some other way, there are some ways to do this. To give an example, I believe vanilla has like two or three ways to set the item count of a stack in a loot table.

This mostly benefits using nuggets (or similar) as output and bonuses.

**Comment by Yefancy** [comment]
you can achieve it by add multiple output ingrredients with chance

**Comment by Kevadroz** [comment]
> you can achieve it by add multiple output ingrredients with chance

Yeah, but that's not really a good way of doing that, wouldn't show up nicely in a recipe viewer and probably wouldn't be done at all in an actual modpack, at least with more than let's say four max items.
